.ci/schedule-weekly.groovy

```diff
@@ -66,6 +66,13 @@ pipeline {
         )
       }
     }
+    stage('Stalled Beats Bumps') {
+      steps {
+        notifyStalledBeatsBumps(branch: 'master', sendEmail: false, to: 'beats-contrib@elastic.co')
+        notifyStalledBeatsBumps(branch: '8.0', sendEmail: false, to: 'beats-contrib@elastic.co')
+        notifyStalledBeatsBumps(branch: '7.16', sendEmail: false, to: 'beats-contrib@elastic.co')
```

Comment:
Requires https://github.com/elastic/apm-pipeline-library/pull/1412 to move away from hardcoded to dynamic branch detection